### PR TITLE
fix(RELEASE-1304): invalid result value

### PR DESCRIPTION
**Pull request by mmalina**
It turns out Tekton only supports passing task results as pipeline results, so we need to pass the PLR name to the task first and then emit it from the task.

### CODE REVIEW

**Review by johnbieren — DISMISSED** [review]

**Review by scoheb — DISMISSED** [review]

**Comment by johnbieren** [comment]
I propose promoting this to staging once merged. The only other commit we would be fast promoting would be the jq default bug fix, and if we don't, this pipeline will go to prod broken eventually

**Comment by mmalina** [comment]
> I propose promoting this to staging once merged. The only other commit we would be fast promoting would be the jq default bug fix, and if we don't, this pipeline will go to prod broken eventually

Agreed, exactly what I was gonna propose :)

**Comment by mmalina** [comment]
I will merge this once task tests are finished. e2e does not test internal resources. The task test actually confirms that this should work in the pipeline as well - it passes the exact same context variable.

**Pull request merged**
